documents/operating.md

```diff
@@ -52,14 +52,202 @@ SECTION 4: 💵💵💵 MONEY 💵💵💵
 ---------------------------------------------------------------------
 SECTION 5: 🗣🗣🗣 Governing Process 🗣🗣🗣
 ---------------------------------------------------------------------
-- [ ] Types of decision
-- [ ] Records and documentation
-- [ ] Membership
-- [ ] Mechanisms for deliberation
-  - [ ] ... On GitHub
-  - [ ] ... On Slack
-- [ ] Use of emoji
-- [ ] Meetings
+- [x] Types of decision
+- [x] Records and documentation
+- [x] Mechanisms for deliberation
+  - [x] ... On GitHub
+  - [x] ... On Slack
+  - [x] ... Plotting Sessions
+- [x] Use of emoji
+
+### Deliberation Processes
+
+#### Slack
+Decisions made over slack are intended to be short term decisions that
+do not require intense deliberation or buy-in from the majority of the
+Hive Mind.
+
+1. A single pinned message is created, posing the entire question up
+for vote.
+
+2. Discussion happens entirely in a threaded response to the question.
+
+3. Voting is performed using emoji in response to a single pinned message
+that poses the entire content up for vote. Positive Emoji are used for a yes
+vote, Negative Emoji are used for a no vote, and Ambiguous Emoji are used
+for an abstention.
+
+4. The results of a vote are tallied and reported back inside of the thread
+in addition to being catalogued in `/votes/slack` as described later in this
+document.
+
+
+#### Plotting Session
+Plotting sessions are agenda driven and require a Quorum to occur.  They
+are led by the chair (as determined by the chain of command).
+
+The Hive Mind will use Robert's Rules of Order to run plotting sessions
+in an orderly and Robertly fashion.
+
+If the plotting session is held virtually, video should be enabled for all
+attendants where bandwidth allows.
+
+If the virtual platform enables costumes or static overlays on top of avatars,
+the meeting chairperson is to select an overlay to represent the Greek Chorus.
+This overlay should be selected to maximally hide the faces of the Greek Chorus.
+
+Each Corporate Overlord must wear overlays depicting wealth.  For example this
+could be a monacle, crown, or in the distant future a laser that fires precious
+gems.
+
+If the virtual platform enables emoji or other forms of visual flair, votes are
+to be calcualted using the platform's equivalent of a thumbs up or thumbs down
+emoji. If directly equivalent emoji reactions do not exist on the platform, the
+meeting chair may select alternative emoji to represent positive and negative votes.
+
+If there is no support for emoji or appropriate effects the Corporate Overlords must
+instead fabricate physical emoji on sticks for both positive and negative votes.  If an
+overlord is unable to fabricate an emoji, they must draw a shaded thumbs up and a
+thumbs down on their hand using a marker or pen.  If they don't have hands or a means
```

Comment:
Please can we just use our literal hands if we can't make the emoji. Please, I'm so tired.